certbot/renewal.py

```diff
@@ -38,7 +38,7 @@
                     "standalone_supported_challenges", "renew_hook",
```

Comment:
I missed this in my standalone PR but I think we should rip out all of the `--standalone-supported-challenges` code including it being included in the list here. Why?

1. It's been deprecated for forever.
2. With the changes we made in your other PR, it will suddenly blow up on TLS-SNI-01 values.
3. The flag is now meaningless since as of writing this standalone only supports HTTP-01.

I don't really care if this is done in this or another PR, but I think we should definitely do this before our next release.

Reply:
Removing an old deprecated and useless argument, and reduce the technical debt? I am your man.